drivers/mpsl/flash_sync/flash_sync_mpsl.c

```diff
@@ -140,9 +140,17 @@ void nrf_flash_sync_set_context(uint32_t duration)
 	_context.request_length_us = duration;
 }
 
+static bool is_in_fault_isr(void)
+{
+	uint32_t isr;
+
+	isr = __get_IPSR();
+	return (isr >= 3 && isr <= 6);
```

Comment:
please do not hardcode numbers here, define 3 and 6, or do something like

```
int32_t irqn = __get_IPSR() - 16;

return (irqn >= HardFault_IRQn && irqn <= UsageFault_IRQn);
```

Reply:
Magic numbers.

Reply:
Fixed